framework/src/modules/interoperability/constants.ts

```diff
@@ -19,6 +19,7 @@ export const MODULE_NAME_INTEROPERABILITY = 'interoperability';
 // General constants
 export const MAINCHAIN_ID = 1;
 export const MAINCHAIN_ID_BUFFER = utils.intToBuffer(MAINCHAIN_ID, 4);
+export const TOKEN_ID_LSK = Buffer.from([0, 0, 0, 1, 0, 0, 0, 0]);
```

Comment:
I think it should be `000000000000` after chainID but maybe we should wait for Token LIP update on this one. Please make a comment as `TODO: To be updated after token module update `